spec/support/shared_examples/name_with_language.rb

```diff
@@ -1,16 +1,16 @@
 shared_examples 'name_with_language' do
 
   describe "#current_language" do
-    it "returns name_zh_tw for chinese" do
-      in_locale 'zh-tw' do
-        expect(described_class.new("test").name__sql).to eq("name_zh_tw")
-      end
-    end
+    # it "returns name_zh_tw for chinese" do
```

Comment:
not working?

Reply:
This was used to test the language in postgres_ext_serializer __sql method
Since we removed that, I think we can delete this file as well